core/src/main/java/com/adobe/aem/guides/wknd/core/models/package-info.java

```diff
@@ -13,7 +13,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-@Version("1.0")
+@Version("2.1.0")
```

Comment:
Just mindlessly copying from tutorial https://experienceleague.adobe.com/docs/experience-manager-learn/getting-started-wknd-tutorial-develop/project-archetype/custom-component.html?lang=en
```
Since the WKND source’s Java™ package com.adobe.aem.guides.wknd.core.models declares version of 1.0.0, and a non-breaking public interface and methods are being added, the version must be increased to 1.1.0. Open the file at core/src/main/java/com/adobe/aem/guides/wknd/core/models/package-info.java and update @Version("1.0.0") to @Version("2.1.0").
```
:D